Containerfile

```diff
@@ -0,0 +1,26 @@
+# Start rust build process
+FROM docker.io/library/rust:latest as builder
+
+# Set working directory to /src, to keep things clean
+WORKDIR /src
+
+# Copy source files to /src
+COPY . /src
+
+# Build
+RUN cargo install --path=./ && cargo clean -v
```

Comment:
This should be `cargo build`, and theres no need for cargo clean as you use a separate container for runtime anyway.

Also you need to run `trunk build` before `cargo build` for the frontend.